mender-convert-package

```diff
@@ -200,10 +205,6 @@ else
     log_fatal "Could not determine root file-system type. Aborting..."
 fi
 
-# NOTE: Since the data partition has a dependency on the rootfs.img checksum
-# when writing the bootstrap Artifact we have to move the datadir out of the
-# way, and write the rootfs image first.
-run_and_log_cmd "mv --force work/rootfs/data work/data"
 
```

Comment:
Please remove the extra newline which is now present here (our shell format checker is complaining about it).

Reply:
Does the commit to fix this also need to follow the "fix" and "Changelog" convention?

Reply:
Yes. But please just amend the original commit, and force-push, don't make a new one.